src/main/java/opendota/Parse.java

```diff
@@ -306,7 +307,18 @@ public void onChatEvent(Context ctx, CDOTAUserMsg_ChatEvent message) {
         entry.value = value;
         output(entry);
     }
-    
+
+    // New chat event
+    @OnMessage(CDOTAUserMsg_ChatMessage.class)
+    public void onAllChatMessage(Context ctx, CDOTAUserMsg_ChatMessage message) {
+        String type = String.valueOf(message.getChannelType());
```

Comment:
What are the possible types here? If we want to be backward compatible I think we have to map 11 to "chat"